doc/man7/EVP_PKEY-ML-KEM.pod

```diff
@@ -16,37 +16,65 @@ EVP_PKEY-ML-KEM
 The B<ML-KEM-512>, B<ML-KEM-768>, and B<ML-KEM-1024> keytypes are implemented
 in OpenSSL's default and FIPS providers.
 
-=for comment (TODO(ML-KEM): Add FIPS support).
-
 =head2 Keygen Parameters
 
-By default, no parameters are required for generating a key pair.
+No mandatory parameters are required for generating a key pair.
 
 =over 4
 
 =item "seed" (B<OSSL_PKEY_PARAM_ML_KEM_SEED>) <octet string>
 
 Internally, ML-KEM generates keys using a 64-byte random value (seed), which is
 the concatenation of the 32-byte I<d> and I<z> parameters described in FIPS 203.
-The optional parameter can be used to set a pre-determined seed prior to
+This optional parameter can be used to set a pre-determined seed prior to
 keypair generation.
-According to FIPS 203, section 3.3, this parameter should only be used for test
-purposes and be treated with the same care as private key material.
 
-This parameter is only settable.
+Generated keys default to retaining the seed used.
+The seed is also by default retained when keys are loaded from B<PKCS#8> files
+in the seed format.
+When available, the seed parameter is also used during key export and import,
+with keys regenerated from the seed even if separately provided on import.
+
+When the seed is retained, it is also available as a B<gettable> parameter,
+and private key output to B<PKCS#8> files will be in seed format.
+When the seed is not available, because not included in the B<PKCS#8> file, not
```

Comment:
because it is not included?

Reply:
"was not included", but I do think the above is idiomatic.